preprolanguage/language/src/main/java/com/oracle/truffle/prepro/runtime/types/PreProConstant.java

```diff
@@ -90,7 +91,7 @@ public PreProVector3 div(PreProVector3 right) {
     @Override
     @TruffleBoundary
     public String toString() {
-        return timeSeries().shapeInfoToString();
+        return Double.toString(getDoubleValue());
```

Comment:
@orat:
Bei den To-String Methoden wird derzeit nur der Rang und die Besetzung der Arrays ausgegeben, nicht die eigentlichen Inhalte.
Für die Konstante könnte es einfacher sein, hier den konkreten Wert zu haben.
Gab es spezielle Gründe, hier nur die Form auszugeben?

Reply:
Ich nehme an, dass die toString() Methode nur fürs Debuggen verwendet wurde. Dabei sind ja Probleme mit Range etc. oft das Problem. Vielleicht sind deshalb diese Infos ausgegeben worden. Ich weiss gar nicht so genau für was die toString()-method alles verwendet wurde. Im Moment sehe ich da keine Schwierigkeiten damit das bei Bedarf zu ändern.